### ISSUE TITLE
finetune the Docformer

### ISSUE THREAD

**Issue by jack-gits** [shabie/docformer#42]
whether I can fine-tune the model of Docformer?
can you give some instruction please. thanks

**Comment by uakarsh** [comment]
You can find the examples [here](https://github.com/shabie/docformer/tree/master/examples/docformer_pl)

**Comment by jack-gits** [comment]
how about this one https://github.com/shabie/docformer/blob/master/examples/DocFormer_for_MLM.ipynb.
what's the difference with above link.

**Comment by jack-gits** [comment]
Docformer is based on Microsolft/LayoutLM, whether we can use it for commercial purpose?

**Comment by uakarsh** [comment]
> how about this one https://github.com/shabie/docformer/blob/master/examples/DocFormer_for_MLM.ipynb. what's the difference with above link.

It is just a pre-training strategy, however you mentioned about fine-tuning, so I shared the same.

I think, that recently LayoutLM where allowed for commercial purpose. You can search for it online.

**Comment by jack-gits** [comment]
the liscense of layoutlmv3 has been changed back. curiously.

**Comment by jack-gits** [comment]
![image](https://user-images.githubusercontent.com/30545972/191015937-367d3f21-9e49-4afc-a759-7e7abd845201.png)

if use_ocr=False, I can't encode the label. there's only have words and boxes in the input para.
how to deal with the labels?

**Comment by uakarsh** [comment]
> ![image](https://user-images.githubusercontent.com/30545972/191015937-367d3f21-9e49-4afc-a759-7e7abd845201.png)
> 
> if use_ocr=False, I can't encode the label. there's only have words and boxes in the input para. how to deal with the labels?

I had earlier shared the link for using DocFormer for token classification, you can visit it and use it for your your own purpose.
